priya/HTML/PriyaAssign9_selectors.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>combinatores</title>
+    <style>
+      .clsdiv {
+        width: 800px;
+        height: 400px;
+        border: 1px solid black;
+        margin: auto;
+        padding: 50px;
+      }
+      div > div:hover {
+        background-color: rgb(206, 203, 203);
+      }
+      span {
+        color: mediumpurple;
+        font-style: italic;
+      }
+      div > div > div {
+        color: mediumseagreen;
+        text-transform: capitalize;
+      }
+      div > div > div::first-letter {
+        color: orange;
+        font-size: xx-large;
+      }
+    </style>
+ </head>
+ <body>
+    <h2><center>Achievers</center></h2>
+    <div class="clsdiv">
+      <div>
+        <h2>
+          Mountineer <span>Poona maalvath</span>,19,is conquering stereotypes
+        </h2>
+        <div>
```

Comment:
make: **font-variant: small-caps in css**

Reply:
ok